.github/workflows/testing.yml

```diff
@@ -41,6 +41,12 @@ jobs:
       - name: Run ShellCheck
         uses: ludeeus/action-shellcheck@master
 
+      - name: Run Swagger Test
```

Comment:
maybe
```suggestion
      - name: Run OpenAPI Test
```

Reply:
The worker is for swagger, that's why I wanted to call it swagger. But I've changed now